script.module.trakt/addon.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.trakt"
        name="trakt.py"
-       version="3.0.0"
-       provider-name="Fuzeman, Razzeee">
+       version="3.1.0"
+       provider-name="Dean Gardiner, Razzeee">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
-    <import addon="script.module.requests" version="2.12.4"/>
-    <import addon="script.module.six" version="1.9.0"/>
-    <import addon="script.module.arrow" version="0.6.0"/>
+    <import addon="script.module.requests" version="2.15.1"/>
```

Comment:
Upstream specifies `2.4.0` as minimum version. Why are you now bumping it to an even higher version then before?

Reply:
To get the latest version that we have in gotham

Reply:
So you don't know that version means minimum version and that the repo for a specific Kodi version contains only one version of every add-on.

Reply:
Sure, but why not force it to get the latest with all possible bugfixes?

Reply:
Changed